extensions/inquiries

```diff
@@ -0,0 +1,73 @@
+# Client to client inquiries
+ 
+## Introduction
+ 
+Several of the existing messages sent by CTCP are inappropriate for transmission
+using the [message intents] framework and a PRIVMSG, as they do not represent an
+ordinary message and in some cases (such as TIME) if translated to use an intent
+tag would give the peculiar case where the PRIVMSG has no actual message, all of
+the information being carried in the intent tag.
+ 
+This latter case, while not prohibited by RFC 1459, is nonetheless prohibited by
+several major ircds.
+ 
+## Capability
+ 
+A client MUST request the `inquiry` capability in order to send or receive these
+messages.
+ 
+## INQUIRE
+ 
+Represents an inquiry from one client to another. The format is as follows:
+ 
+    INQUIRE <Target> <Command> [<argument>, ...]
+ 
+Target identifies the entity being inquired of, while Command specifies the kind
+of inquiry. Any arguments may be optional, depending on the command used.
+ 
+## IQRESULT
+ 
+Represents a response to a previous `INQUIRE`
+ 
+    IQRESULT <Target> <Command> [<argument>, ...]
+ 
+The target and command are the sender of the original INQUIRE and the original
+command. The arguments are whatever response is appropriate for the command.
+ 
+## Sending to channels
+
+Clients MAY send INQUIRE messages with channels as the target, subject to normal
+access control for the channel in question. As with PRIVMSG, the server forwards
+the message to all other users in the channel. A client receiving such a message
+SHOULD respond in the same manner as if the inquiry had been sent directly to it.
+
+Clients MUST NOT send IQRESULT messages to channels. Servers MUST respond to any
```

Comment:
Would it not be better to replace `<Target>` with `<Nick>` instead of explicitly rejecting channels?